reference/errors/overview.md

```diff
@@ -18,11 +18,12 @@ Meilisearch uses the following standard HTTP codes for a successful or failed AP
 
 All detailed task responses contain an [`error`](/reference/api/tasks.md#error) field. When a task fails, it is always accompanied by a JSON-formatted error response. Meilisearch errors can be of one of the following types:
 
-| Type              | Description                                                                                                                                                                                                              |
-| :---------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| **`invalid_request`** | This is due to an error in the user input. It is accompanied by the HTTP code `4xx`                                                                                                                                      |
-| **`internal`**        | This is due to machine or configuration constraints. The most common cause is reaching or exceeding hard limits, such as the size of the disk, the size limit of an index, etc. It is accompanied by the HTTP code `5xx` |
-| **`auth`**            | This type of error is related to authentication and authorization. It is accompanied by the HTTP code `4xx`                                                                                                              |
+| Type                  | Description                                                                                                                                                                               |
+| :-------------------- | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **`invalid_request`** | This is due to an error in the user input. It is accompanied by the HTTP code `4xx`                                                                                                       |
+| **`internal`**        | This is due to machine or configuration constraints. It is accompanied by the HTTP code `5xx`                                                                                             |
+| **`auth`**            | This type of error is related to authentication and authorization. It is accompanied by the HTTP code `4xx`                                                                               |
+| **`system`**          | This indicates your system has reached or exceeded its limit for disk size, index size, etc., or the database doesn't have read or write access. It is accompanied by the HTTP code `5xx` |
```

Comment:
Can we rewrite this without using "etc"? Either we list all causes or state a general category ("This indicates your system has reached or exceeded its disk space usage limits)—"etc" is sort of in-between and leaves too much open for interpretation.